add-convo-record-for-every-convo.py

```diff
@@ -0,0 +1,21 @@
+from sqlite3 import DatabaseError
+
+from sqlalchemy import create_engine, insert
+from sqlalchemy.orm import sessionmaker
+
+from config import Config
+from secure_message.repository.database import Conversation, SecureMessage
+
+engine = create_engine(Config.SQLALCHEMY_DATABASE_URI)
+Session = sessionmaker(bind=engine)
+db = Session()
+
+try:
+    subquery = db.query(Conversation.id)
+    distinct_ids = db.query(SecureMessage.thread_id).filter(~SecureMessage.thread_id.in_(subquery)).distinct()
+    for thread_id in distinct_ids:
+        db.execute(insert(Conversation).values({"id": thread_id, "is_closed": "False"}))
```

Comment:
I think it should at least log out the thread id so progress can be monitored, currently it sits at the command prompt until completion.

Reply:
i second this

Reply:
```INFO:config:thread_id=('4f764286-13eb-4c7a-ab66-862e383afb03',) event='Adding the following threads:'```